src/odemis/gui/cont/views.py

```diff
@@ -325,31 +407,48 @@ def _on_chamber_state(self, state):
             # sample have probably nothing in common with this new sample
             self._data_model.stage_history.value = self._data_model.stage_history.value[-1:]
 
+    def _update_ovv(self):
+        """ Update the overview image with the currently active stream. """
+        if self._data_model.streams.value and self.curr_s \
+                    and self.curr_s.image.value is not None:
+            s = self.curr_s
+            img = s.image.value
+            if isinstance(s, acqstream.OpticalStream):
+                self.im_opt = insert_tile_to_image(img, self.im_opt)
+            elif isinstance(s, acqstream.EMStream):
+                self.im_sem = insert_tile_to_image(img, self.im_opt)
+            else:
+                logging.info("Stream %s not added to overview image" % (s))
```

Comment:
`logging` supports passing the inputs for the string formatting as extra arguments, and it's the recommended way. So you can write:
`logging.info("%s not added to overview image as it's not optical nor EM", s)`